monitoring/views.py

```diff
@@ -138,17 +143,30 @@ def post(self, request, *args, **kwargs):
 
                 org_implementing = Organization.objects.filter(name=organization_name).first()
                 if not subproject:
-                    messages.append('Problem to import record #{} : Subproject with Project "{}" and Organization "{}"\
+                    messages.append('Problem to import record #{} : '
+                                    'Subproject with Project "{}" and Organization "{}"\
                         does not exist!'.format(row[0].row, project_name, organization_name))
 
+                birthdate = row[project_cols + 4].value
+                project_entry_date = row[project_cols + 13].value
+                if birthdate and birthdate.find('—') <= -1:
+                    birthdate = datetime.datetime.strptime(birthdate, date_format)
```

Comment:
mi punto aqui era que segun el manual de openxlpy si la celda es tipo fecha, el valor qeu retornara row[project_cols+4].value sera tipo fecha, entonces no hay necesidad de hacer estar parsing.
seria if row[x].is_date: ... o if not...

Reply:
entiendo su punto, y el problema exactamente no es parseo de string a date, sino del formato que viene al que espera la bd